pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/nsx_add_user_rights/guide.fr-fr.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Activer NSX-T dans un Hosted Private Cloud VMware on OVHcloud"
+excerpt: "Découvrez comment ajouter les droits à un utilisateur et aux Datacentres pour NSX-T"
+updated: 2024-05-07
+flag: hidden
```

Comment:
I added this tag as I'm not convinced images integration in the <details> format will work.
This will prevent the guide from being visible at first, letting us bandwidth to fix formats.